.github/workflows/python-package.yml

```
@@ -32,63 +34,35 @@ jobs:
+        env:
+          PYTHONPATH: ${{ github.workspace }}
+      - name: Run lint with nox
+        run: nox -s lint
+        env:
+          PYTHONPATH: ${{ github.workspace }}
+      - name: Run tests with nox
+        run: nox -s tests
+        env:
+          PYTHONPATH: ${{ github.workspace }}
+      - name: Upload coverage reports
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.4)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)